packages/create-webstone-app/src/package.ts

```diff
@@ -0,0 +1,39 @@
+import { PackageJson } from "type-fest";
+
+export const appPackageJson: PackageJson = {
+  devDependencies: {
+    "@webstone/cli": "^0.12.0",
```

Comment:
🤔 I need to think how we could automate this, maybe with dependabot...? It looks like a maintenance burden we will easily forget about and wonder why new apps or plugins depend on an old Webstone CLI